assets/pcw-hugo-theme/css/colors.css

```diff
@@ -0,0 +1,76 @@
+/*********
+* COLORS *
+*********/
+
+:root {
+  --orange: rgb(255, 128, 0);
+
+  --blue: #5262ed;
+  --lilac: #d48aff;
+  --dark-blue: #123248;
+  --light-gray: #e6e6e6;
+
+  --primary: var(--blue);
+}
+
+/* Background color */
+
+.bg-pcw {
+  background-color: var(--primary);
+  color: white;
+}
```

Comment:
Pulled the "colors" section out into this file. Only made one change to this block of text... removing the "!important" property from all of these rules.